oc-game-swift/Game/Game.swift

```diff
@@ -7,3 +7,106 @@
 //
 
 import Foundation
+
+// Create the Game class
+class Game {
+    static var playerList = [Player]()
+    
+    // Welcome Menu
+    static func start() {
+        print("SWIFT TO THE DEATH"
+            + "\n----------------------"
+            + "\n"
+            + "\n Select an option"
+            + "\n"
+            + "\n1. Start New Game"
+            + "\n2. Credits")
+        // If options 1 or 2 are choosen
+        if let line = readLine(){
+            switch line{
+            case "1": // Move on to the player creation
+                Game.createPlayers()
+            case "2":
+                Game.showCredits()
+            default: // If not
+                print("You have to choose 1 or 2 😅")
+                Game.start()
+            }
+        }
+        
+    }
+    
+    // Show credits
+    static func showCredits() {
+        print("This awesome game is developped by Jérôme Krakus !")
+    }
+    
+    // Create players
+    static func createPlayers() {
+        // While there is no 2 players for the game
+        while Game.playerList.count < 2 {
```

Comment:
if you plan to add more users next, maybe you could think about another solution ?